index.js

```diff
@@ -2,73 +2,74 @@ const fs = require("fs");
 const path = require("path");
 const { spawn } = require("child_process");
 const { shell } = require("electron");
-var ncp = require("copy-paste");
+const ncp = require("copy-paste");
 const recording = require("./recording");
-console.log('index.js running')
+const del = require('del');
+
 exports.decorateTerms = (Terms, { React, notify }) => {
   return class extends React.Component {
     constructor(props, context) {
       super(props, context);
       this.terms = null;
       this.onDecorated = this.onDecorated.bind(this);
       this.state = {
-        isRecording: false
+		isRecording: false,
+		filename: `terminal-session-${performance.now()}.webm`

```

Comment:
You only should put stuff in `this.state` that can change at runtime. Since this is a static value, just declare it on the class:

```js
this.fileName = `terminal-session-${performance.now()}.webm`
```